pg_search/src/api.rs

```diff
@@ -22,9 +22,81 @@ pub fn weighted_mean(a: f64, b: f64, weights: Vec<f64>) -> f64 {
     );
 
     assert!(
-        (weight_a + weight_b - 1.0).abs() < std::f64::EPSILON,
+        (weight_a + weight_b - 1.0).abs() < f64::EPSILON,
         "Weights must add up to 1"
     );
 
     a * weight_a + b * weight_b
 }
+
+#[cfg(feature = "pg_test")]
+#[pgrx::pg_schema]
+mod tests {
+    use pgrx::*;
+    use shared::testing::SETUP_SQL;
+
+    use super::{minmax_norm, weighted_mean};
+
+    #[test]
+    fn test_minmax_norm() {
+        let value = 60.0;
+        let min = 20.0;
+        let max = 30.0;
+        assert_eq!(minmax_norm(value, min, max), (value - min) / (max - min));
+    }
+
+    #[test]
```

Comment:
these don't work. We need them to be #[pg_test] everywhere. We got bit by having regular tests after merging your previous PR.

Reply:
No worries at all, we didn't know either!